circle.yml

```diff
@@ -21,19 +24,26 @@ machine:
     PATH: "$HOME/.config/composer/vendor/bin:$PATH"
   hosts:
     local.mysiteurl: 127.0.0.1
+  services:
+    - docker
 
 dependencies:
   cache_directories:
     - ~/.composer/cache
     # Cache DB dumps directory.
     - .data
   pre:
+    # Stopping default mysql service so the db container can expose itself on port 3306.
+    - sudo service mysql stop
     # Download DB dump from public URL to test Drupal-dev project.
     # Remove the line below in your project.
-    - if [ ! -f $BUILD_DIR/.data/db.dist.sql ] ; then mkdir $BUILD_DIR/.data && curl -L https://goo.gl/WFtJbT -o $BUILD_DIR/.data/db.dist.sql; fi
+    - if [ ! -f "${DB_CONTAINER_TAR}" ] ; then mkdir $BUILD_DIR/.data && curl -L https://goo.gl/WFtJbT -o $DB_SOURCE_DUMP; bash <(curl -s "${DB_CONTAINER_SCRIPT}"); fi
```

Comment:
can you please use multiline YAML syntax for this so that end user could easily enable/disable preferred way of DB handling in one place rather then scanning through many lines

```
- |
    if [ ! -f "${DB_CONTAINER_TAR}" ] ; then mkdir $BUILD_DIR/.data && curl -L https://goo.gl/WFtJbT -o $DB_SOURCE_DUMP; bash <(curl -s "${DB_CONTAINER_SCRIPT}"); fi

...
```